<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Table</class>
 <widget class="QWidget" name="Table">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>699</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Table</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_11">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <widget class="QLabel" name="label_title">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>表格</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_current_item">
       <property name="text">
        <string>当前项</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_17">
     <item>
      <widget class="QTableWidget" name="tableWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <row>
        <property name="text">
         <string>row0</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>row1</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>row2</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>col0</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>col1</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>col2</string>
        </property>
       </column>
       <item row="0" column="0">
        <property name="text">
         <string>A1</string>
        </property>
       </item>
       <item row="0" column="1">
        <property name="text">
         <string>B1</string>
        </property>
       </item>
       <item row="0" column="2">
        <property name="text">
         <string>C1</string>
        </property>
       </item>
       <item row="1" column="0">
        <property name="text">
         <string>A2</string>
        </property>
       </item>
       <item row="1" column="1">
        <property name="text">
         <string>B2</string>
        </property>
       </item>
       <item row="1" column="2">
        <property name="text">
         <string>C2</string>
        </property>
       </item>
       <item row="2" column="0">
        <property name="text">
         <string>A3</string>
        </property>
       </item>
       <item row="2" column="1">
        <property name="text">
         <string>B3</string>
        </property>
       </item>
       <item row="2" column="2">
        <property name="text">
         <string>C3</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>输出选中项</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QPushButton" name="pushButton_print_selected">
          <property name="text">
           <string>输出选中项</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="plainTextEdit_print_selected"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_16">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>设置表格</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_set_row_count"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_set_row_count">
            <property name="text">
             <string>设置行数</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_set_col_count"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_set_col_count">
            <property name="text">
             <string>设置列数</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLineEdit" name="lineEdit_set_header">
            <property name="placeholderText">
             <string>表头1,表头2,表头3...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_set_header">
            <property name="text">
             <string>设置表头标签</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QRadioButton" name="radioButton_interactive">
              <property name="text">
               <string>手动调整列宽</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup</string>
              </attribute>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QRadioButton" name="radioButton_fixed">
                <property name="text">
                 <string>固定值列宽</string>
                </property>
                <attribute name="buttonGroup">
                 <string notr="true">buttonGroup</string>
                </attribute>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spinBox_resize_header_value">
                <property name="maximum">
                 <number>200</number>
                </property>
                <property name="value">
                 <number>50</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_stretch">
              <property name="text">
               <string>自动调整列宽-伸展</string>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_resize_to_contents">
              <property name="text">
               <string>自动调整列宽-调整到内容大小</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup</string>
              </attribute>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_set_header_resize">
            <property name="text">
             <string>设置列宽</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>排序</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QCheckBox" name="checkBox_sortable">
             <property name="text">
              <string>允许排序</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_8">
          <property name="title">
           <string>合并单元格</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_10">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_14">
             <item>
              <spacer name="horizontalSpacer_5">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <layout class="QGridLayout" name="gridLayout">
               <item row="0" column="2">
                <widget class="QSpinBox" name="spinBox_span_colspan">
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>999</number>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QSpinBox" name="spinBox_span_col">
                 <property name="maximum">
                  <number>999</number>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QSpinBox" name="spinBox_span_row">
                 <property name="maximum">
                  <number>999</number>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QSpinBox" name="spinBox_span_rowspan">
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>999</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_span">
               <property name="text">
                <string>合并</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_10">
          <property name="title">
           <string>分页</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_18">
             <item>
              <widget class="QCheckBox" name="checkBox_split_page">
               <property name="text">
                <string>允许分页</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_6">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>每页行数：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox_every_page_row_number">
               <property name="minimum">
                <number>5</number>
               </property>
               <property name="maximum">
                <number>1000</number>
               </property>
               <property name="value">
                <number>20</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_split_page">
               <property name="text">
                <string>分页</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_current_page">
             <property name="text">
              <string>当前页码：</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_19">
             <item>
              <widget class="QPushButton" name="pushButton_prev_page">
               <property name="text">
                <string>上一页</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_next_page">
               <property name="text">
                <string>下一页</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_9">
       <property name="title">
        <string>设置数据</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>增 &amp;&amp; 改</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox_add_item_row"/>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox_add_item_col"/>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_7">
                 <item>
                  <widget class="QLineEdit" name="lineEdit_add_item_content"/>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pushButton_add_item">
                   <property name="text">
                    <string>赋值</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_8">
                 <item>
                  <widget class="QLineEdit" name="lineEdit_background_color">
                   <property name="text">
                    <string>#00B294</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pushButton_background_color">
                   <property name="text">
                    <string>背景色</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_9">
                 <item>
                  <widget class="QLineEdit" name="lineEdit_foreground_color">
                   <property name="text">
                    <string>#FF0000</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pushButton_foreground_color">
                   <property name="text">
                    <string>前景色</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_add_A1_to_Z26">
             <property name="text">
              <string>初始值，A1-Z26</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_add_1_to_million">
             <property name="text">
              <string>初始值，1-1,000,000</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_7">
          <property name="title">
           <string>删</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_13">
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox_remove_row"/>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox_remove_col"/>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_remove">
               <property name="text">
                <string>删除</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_15">
          <item>
           <widget class="QGroupBox" name="groupBox_6">
            <property name="title">
             <string>内容匹配规则</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_12">
             <item>
              <widget class="QRadioButton" name="radioButton_equal">
               <property name="text">
                <string>精确匹配</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButton_include">
               <property name="text">
                <string>包含*</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButton_start_with">
               <property name="text">
                <string>以*开头</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButton_end_with">
               <property name="text">
                <string>以*结尾</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButton_ignore_capltals">
               <property name="text">
                <string>忽略大小写</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButton_regular_expression">
               <property name="text">
                <string>正则匹配</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_5">
            <property name="title">
             <string>查</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_12">
               <item>
                <widget class="QLineEdit" name="lineEdit_find_match"/>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_find_match">
                 <property name="text">
                  <string>内容匹配查找</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox_highlight_find_result">
               <property name="text">
                <string>标红查找结果</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox_scroll_to_find_match">
               <property name="text">
                <string>自动跳转到查找结果</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPlainTextEdit" name="plainTextEdit_find_match">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>1</verstretch>
                </sizepolicy>
               </property>
               <property name="plainText">
                <string>依内容查找 匹配结果</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
